frontend/src/components/ui/toast.tsx

```diff
@@ -92,7 +92,7 @@ const ToastTitle = React.forwardRef<
 >(({ className, ...props }, ref) => (
   <ToastPrimitives.Title
     ref={ref}
-    className={cn("text-sm font-semibold [&+div]:text-xs", className)}
+    className={cn("break-all text-sm font-semibold [&+div]:text-xs", className)}
```

Comment:
we should be careful with using `break-all`. Can you try `break-words`?

Reply:
Yup was also concerned about "more" being broken into "m-ore". But `break-words` doesn't do the job, the problem is caused by one word (the pubkey)